rosdep/python.yaml

```diff
@@ -7018,6 +7018,9 @@ python3-websocket:
   openembedded: ['python3-websockets@meta-python']
```

Comment:
Can you please update this to ['python3-websocket-client@meta-python']
https://layers.openembedded.org/layerindex/recipe/97740/
I didn't realize that websocket and websockets are 2 different packages - or I can fix it my next PR as it's used also in python-websocket mapping above.

Reply:
Done in 3102f3c.

Reply:
@shr-project @Goldob I would not make changes to unrelated rules. 

To modify `python-websocket` and `python3-websocket`, please open a different PR.

Reply:
I'm sorry @Goldob

separate PR for OE websocket is already created https://github.com/ros/rosdistro/pull/27826 just a draft now, waiting to collect more changes before submitting it.

Reply:
No worries @shr-project. I reverted the changes.